CONTRIBUTING.md

```diff
@@ -163,6 +163,8 @@ was broken. They are slower than the linters but will typically catch more error
 ```sh
 source ./env/bin/activate
 trial tests
+# The tests can also be run with this longer command
+python -m twisted.trial tests
```

Comment:
I think it's good to call this out because the readme uses this syntax and people might be Ctrl+F finding where to jump off from.

Reply:
I see your point, though I'm bound to say that I think this just makes it confusing for readers of CONTRIBUTING.md.

A better solution would be to use one command or the other consistently. (I'd suggest `trial` since it's shorter, but ymmv)

Reply:
I've updated the readme to use `trial tests` 🤷